wfe/wfe.go

```diff
@@ -627,6 +627,18 @@ func (wfe *WebFrontEndImpl) NewRegistration(ctx context.Context, logEvent *web.R
 
 	reg, err := wfe.RA.NewRegistration(ctx, init)
 	if err != nil {
+		if berrors.Is(err, berrors.Duplicate) {
```

Comment:
This is a nice improvement over the code we already have on line 590-598, since it will work more reliably. Can you delete the preexisting code since this should cover that case adequately?

Reply:
I'm not concretely against this, but it does introduce some weird corner cases. For instance if we change our key requirements a user will still be able to use a key that doesn't match those requirements, but if they try to use the NewRegistration endpoint to find their account URI it'll fail out with an error saying their key is no okay to use, despite it already being in use.